File: SofiForce.Models/Models/ListModels/OperationRequestDetailPreferredTimeListModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class OperationRequestDetailPreferredTimeListModel
    {

        public string Id { get; set; }
        public long PreferredId { get; set; }
        public long? DetailId { get; set; }
        public int? PreferredOperationId { get; set; }
        public int? WeekDayId { get; set; }
        public string FromTime { get; set; }
        public string ToTime { get; set; }
        public string WeekDayName { get; set; }
        public string PreferredOperationName { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/OperationRequestDetailPreferredTimeListModel.ts
export interface OperationRequestDetailPreferredTimeListModel {
    preferredId: number;
    detailId: number | null;
    preferredOperationId: number | null;
    weekDayId: number | null;
    fromTime: string | null;
    toTime: string | null;
    weekDayName: string;
    preferredOperationName: string;
}